packages/plugins/prometheus/src/index.ts

```diff
@@ -183,17 +183,21 @@ export function usePrometheus(options: PrometheusTracingPluginConfig): Plugin {
     onRequest({ request }) {
       startByRequest.set(request, Date.now());
     },
-    onParse() {
-      return ({ result: document, context }) => {
-        const operationAST = getOperationAST(document, context.params.operationName);
-        const params = {
-          document,
-          operationName: operationAST?.name?.value,
-          operationType: operationAST?.operation,
-        };
+    onParse({ context }) {
+      // If only it is Yoga, we calculate HTTP request time
+      if (context.request) {
```

Comment:
I'm not sure to understand why we don't want to also track subscriptions ?

Reply:
Subscriptions are not the case here. Yoga subscriptions(SSE) are still tracked.
The issue is when GraphQL WS uses the Envelop instance of Yoga, this plugin fails because GraphQL WS doesn't have `request` object in the context.
It is ok to skip this because GraphQL WS is not HTTP at all.
This doesn't skip tracking entirely, it just skips the metric starts on `onRequest` and ends in `onResponse` hooks. Those hooks are not triggered by GraphQL WS at all.